tests/main/system-usernames-illegal/task.yaml

```diff
@@ -3,7 +3,7 @@ summary: ensure unapproved user cannot be used with system-usernames
 # List of expected snap install failures due to libseccomp/golang-seccomp being
 # too old. Since the illegal name check happens after verifying system support,
 # we can ignore these.
-systems: [-amazon-linux-2-*, -centos-7-*, -debian-9-*, -fedora-29-*, -fedora-30-*, -opensuse-15.2-*, -opensuse-15.1-*, -ubuntu-14.04-*]
+systems: [-amazon-linux-2-*, -centos-7-*, -debian-9-*, -debian-10-*, -fedora-29-*, -fedora-30-*, -opensuse-15.2-*, -opensuse-15.1-*, -ubuntu-14.04-*]
```

Comment:
It is unexpected to see debian 10 here, we need to investigate.

Fedora 32 is the oldest release we have in spread.yaml. I think we can garbage collect fedora-(29|30) here.